<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="5CD6474QH2" ignored="0" tests="1" failures="1" timestamp="21 ноя 2018 12:15:32 GMT" time="15.56" errors="0">
  <testcase name="testingActionTest" time="4.933" classname="StepTest">
    <failure type="java.lang.AssertionError" message="expected [5] but found [3]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [3]
at StepTest.testingActionTest(StepTest.java:41)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testingActionTest -->
</testsuite>
